### DISCUSSION TITLE
Self host on Ubuntu could not access from outside

### DISCUSSION THREAD

**Discussion by dzungpv** [open-meteo/open-meteo#376] [Q&A] [answered]
I set up a server with Ubuntu jammy packages but it could not access outside.
I use Digital Ocean server, when I stop the openmeteo-api service and run command:
```
openmeteo-api serve 
```
then it can access from outside.
So how can I use automatics startup service to server the API?

**Comment by dzungpv** [comment, marked as answer]
I solve it myself , edit file /etc/default/openmeteo-api.env and change 
API_BIND="127.0.0.1:8080" to API_BIND="0.0.0.0:8080"
So please update the docs.

**Reply by patrick-zippenfenig** [reply]
Thanks for sharing. I added it to the docs: https://github.com/open-meteo/open-meteo/commit/36e8e53998405c594f10167a481957b2c75d925c


Note: I would recommend to use NGINX in front of the API instance

**Reply by dzungpv** [reply]
How to do it? You mean setup a proxy with NGINX?

**Reply by patrick-zippenfenig** [reply]
Yes, see here https://docs.vapor.codes/deploy/nginx/#configure-proxy

**Reply by dzungpv** [reply]
Thank you. Off topic But I am confuse with parameter: "run 00" or "run 03" "run 2023062500" so it is timestamp?

**Reply by patrick-zippenfenig** [reply]
It can be both. Usually it should only be the initialisation hour `00` or `03`, but you can also specify a date + hour. I added the later functionality, to redownload certain runs in case of failure.
